tests/rs_sqla_test_utils/models.py

```diff
@@ -88,6 +88,7 @@ class ReflectionIdentity(Base):
     __tablename__ = 'reflection_identity'
     col1 = sa.Column(sa.Integer(), primary_key=True)
     col2 = sa.Column(sa.Integer(), info={'identity': (1, 3)})
+    col3 = sa.Column(sa.Integer())
```

Comment:
This test was only passing before because the IDENTITY column already happened to be the last one. This test now fails under the old code and passes under the new code (where we add `attnum` to the ORDER BY clause).